howso/details.amlg

```diff
@@ -71,7 +71,7 @@
 	;						model area. Uses only the context features of the reacted case to determine that area.
 	;
 	;		 "categorical_action_probabilities" true or false. If true outputs probabilities for each class for the action
-	;						features. Applicable only to categorical action features. react_series() calls additionally
+	;						features. Applicable only to categorical action features. single_react_series() calls additionally
```

Comment:
can we drop the parenthesis here? these definitely don't make sense here in Amalgam land